fcm.go

```diff
@@ -166,6 +167,7 @@ func (this *FcmClient) apiKeyHeader() string {
 func (this *FcmClient) sendOnce(ctx context.Context) (*FcmResponseStatus, error) {
 	fcmRespStatus := new(FcmResponseStatus)
 
+	this.Message.DirectBootOk = true
```

Comment:
we shouldn't embed this in the package like this. Let's leave it to the calling context to call `SetDirectBootOk()`

Reply:
ok. Updated